olive/passes/utils/whisper_prepost.py

```diff
@@ -328,6 +329,18 @@ def add_pre_post_processing_to_model(
     testdata_filepath: str,
     use_audio_decoder: bool = True,
 ) -> onnx.ModelProto:
+    # TODO: support onnxruntime-extensions version >= 0.9.0
+    # once 0.9.0 is supported, we probably cannot support < 0.9.0 since there are large changes in the export
+    # refer to this commit https://github.com/microsoft/onnxruntime-extensions/commit/62d8598
+    # check onnxruntime-extensions version
+    from onnxruntime_extensions import __version__ as ort_ext_version
+
+    if version.parse(ort_ext_version) != version.parse("0.8.0"):
```

Comment:
I am guessing the check is needless. by looking at the error from internet, it seems the model could still be inference even if the check_model failure.

Reply:
Will close this PR. Will check for compatability with the newest version later when adding support for 0.9.0